src/main/java/meditracker/storage/FileReaderWriter.java

```diff
@@ -119,4 +130,58 @@ public static void saveMediTrackerData(MedicationManager medManager) {
             System.out.println(e.getMessage());
         }
     }
+
+    /**
+     * Loads the daily medication information from a fixed file data/dailymed/today.txt.
+     * The loading functionality will be improved in v2.0.
+
+     * @param dailyMedData A list of type String for the daily medication data.
+     * @throws FileReadWriteException if there is an issue creating the file.
+     */
+    public static void saveDailyMedicationData(List<String> dailyMedData) throws FileReadWriteException {
+        initialiseDirectory(getDailyMedicationFolderName());
+
+        // This part is similar to `createJsonSaveFile()`
+        File fileToWrite = new File(getFullDailyMedFilePath());
+        try {
+            fileToWrite.delete();
+            fileToWrite.createNewFile();
+        } catch (IOException e) {
+            throw new FileReadWriteException("IO Error: Unable to write to JSON File");
+        } catch (SecurityException e) {
+            throw new FileReadWriteException("Unable to create save JSON file. Please make sure that "
+                    + "the file has the appropriate permissions for MediTracker to write to.");
+        }
+
+        //@@author annoy-o-mus-reused
+        // Reused from https://stackoverflow.com/a/6548204
+        // with minor modifications
+        try {
+            FileWriter f = new FileWriter(fileToWrite);
+            for (String stringData : dailyMedData) {
+                f.write(stringData + System.lineSeparator());
+            }
+            f.flush();
+            f.close();
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    /**
+     * Saves the daily medication information to a fixed file data/dailymed/today.txt.
```

Comment:
Just like the comment above, should this be "Loads the ..." as the method is loadDailyMedicationData() ?

Reply:
See above comment